main.py

```diff
@@ -20,107 +21,140 @@ class DnsInfo(TypedDict):
     modified_on: str
 
 
-def list_zones() -> list[DnsInfo]:
-    '''List all zones associated with this cloudflare API key'''
+def list_zones() -> List[DnsInfo]:
+    """list_zones.
 
-    endpoint = 'zones'
+    List all zones associated with this cloudflare API key
```

Comment:
There's a few "docstring templates" that are generally used - google and numpy are the top two options. I like both, I configured vim to give me google style automatically based on the args/return values

Reply:
This is a good tip, but it bothers me to have the docstring list the parameters and return type when this is already stated in the function definition.

Reply:
Hmm I never really considered that because I auto generate docstrings but I can see how that is like doing the same thing twice. Numpy format does it too... someone must have a good explanation but I do not